### PR TITLE
feat: adding implementation and tests for resolving scss include paths

### PR DESCRIPTION
**Pull request by mwootendev**
## I'm submitting a...

```
[ ] Bug Fix
[X] Feature
[ ] Other (Refactoring, Added tests, Documentation, ...)
```

## Checklist

- [X] Commit Messages follow the [Conventional Commits](https://conventionalcommits.org/) pattern
  - A feature commit message is prefixed "feat:"
  - A bugfix commit message is prefixed "fix:"
- [X] Tests for the changes have been added

## Description

Updates to the configuration under the `lib` element to add a new `sassIncludePaths` option, which takes an array of paths that will be used to resolve SCSS `@import` statements by the SASS compiler. If the paths are not absolute, they will be resolved relative to the ngPackage configuration file.

* Updated ng-package.schema.json to include the new option.
* Updated NgEntryPoint to read the property and resolve URLs relative to the package.
* Updated the assets step to pass the included paths to the SASS compiler.
* Added a new scss-paths integration test suite to ensure the SASS imports are working properly.
* Updated the README to include a new section on how to use the option.

To use the new option, users will add a `sassIncludePaths` section in the ngPackage configuration:

```json
{
  "ngPackage": {
    "lib": {
      "sassIncludePaths": ["./src/assets/styles"]
    }
  }
}
```

This pull request would resolve Issue #282 by adding the ability to include additional paths when resolving SCSS files from component imports.

## Does this PR introduce a breaking change?

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Comment by cacothi** [comment]
Hi guys,

I believe my issue is the same. I do want also to share the Stylesheet between projects. So not only the component would be shared but also all the scss files could be consumed by the other projects.

Any update on this feature?

File: README.md [hunk 1]
```diff
@@ -222,6 +222,21 @@ Valid values: `none` or `inline`.
 }
 ```
 
+#### What if I have multiple SASS/SCSS include paths?
+
+In case you have multiple include paths for `@import` statements (e.g., when setting the `stylePreprocessorOptions` in `.angular-cli.json`),
+the additional paths may be configured through the the `sassIncludePaths` option.
```
Comment (dherges) [inline README.md]: I added some formatting to the markdown.

File: src/ng-package.schema.json [hunk 2]
```diff
@@ -64,6 +64,13 @@
           ],
           "default": "none"
         },
+        "sassIncludePaths": {
+          "description": "Any additional paths that should used to resolve SASS imports",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
```
Comment (dherges) [inline src/ng-package.schema.json]: :+1:

File: integration/samples/scss-paths/sub-module/package.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  "private": true,
+  "repository": "https://github.com/dherges/ng-packagr.git",
+  "peerDependencies": {
+    "@angular/core": "^4.1.2",
+    "@angular/common": "^4.1.2"
+  },
+  "ngPackage": {
+    "lib": {
+      "entryFile": "index.ts",
+      "sassIncludePaths": ["../assets","./assets"]
```
Comment (dherges) [inline integration/samples/scss-paths/sub-module/package.json]: Ah, yes, looks good!

**Comment by dherges** [comment]
Hi @mwootendev,

thanks for your time and effort! Looks good to me!

**Pull request merged**